### PR TITLE
Reword vector layer capability strings

### PR DESCRIPTION
**Pull request by strk**
Clarifies that not all capabilities are "editing capabilities"
and that "Simplify Geometry" has nothing to do with "Change Geometry"

Who's on the UX committee to review this ?
@jef-n, @m-kuhn ?

### CODE REVIEW

**Comment by strk** [comment]
@slarosa, @nyalldawson (as I saw you posting on qgis-ux list ;0)

**Comment by nyalldawson** [comment]
Looks ok to me. There's a bunch of inconsistent capitalisation throughout those strings which would be good to fix to.

**Comment by strk** [comment]
should "before fetch the feature" also be "before fetching the feature" ?

**Comment by m-kuhn** [comment]
As also a non-native speaker, I think yes.
And maybe it should be "Presimplify Geometries *while* keeping them valid" instead of "Presimplify Geometries keeping them valid"?

**Comment by strk** [comment]
@m-kuhn could you please merge this if happy ?

**Comment by m-kuhn** [comment]
I'd rather have a native speaker take care

File: src/core/qgsvectordataprovider.cpp [hunk 1]
```diff
@@ -204,20 +204,20 @@ QString QgsVectorDataProvider::capabilitiesString() const
...
-    QgsDebugMsg( "Capability: Simplify Geometries before fetch the feature ensuring that the result is a valid geometry" );
+    abilitiesList += tr( "Presimplify Geometries while keeping them valid" );
+    QgsDebugMsg( "Capability: Simplify Geometries before fetching the feature ensuring that the result is a valid geometry" );
   }
 
   if ( abilities & QgsVectorDataProvider::ChangeFeatures )
   {
     abilitiesList += tr( "Joint Geometry and Attributes updates" );
-    QgsDebugMsg( "Capability: change attributes and geometries at once" );
+    QgsDebugMsg( "Capability: Change Attributes and geometries at once" );
```
Comment (m-kuhn) [inline src/core/qgsvectordataprovider.cpp]: attributes should probably not be capitalized (like geometries isn't either)

Comment (strk) [inline src/core/qgsvectordataprovider.cpp]: As this PR started to introduce some "consistency" I copied the
approach taken in all other labels to capitalize the first two words.

I don't like it either, FWIW ..

Comment (m-kuhn) [inline src/core/qgsvectordataprovider.cpp]: It just seems weird to have Attribute capitalized and geometries not capitalized just next to each other. Wouldn't it be better to switch the other to lower case instead?

Comment (strk) [inline src/core/qgsvectordataprovider.cpp]: I'll leave this to someone else to decide.
To me even the "Capability: " prefix is superfluous.

Comment (m-kuhn) [inline src/core/qgsvectordataprovider.cpp]: More important than the debug messages would be the user-facing abilitiesList. @nyalldawson , @NathanW2 , @timlinux (or any other native speaker).

**Comment by nyalldawson** [comment]
I've pushed a modified version as cd145bb4b3f08364d459b6bf8b512a3891764400

**Comment by strk** [comment]
Thanks @nyalldawson

**Pull request closed without merging**